### PR TITLE
Add junit test dependency

### PR DESCRIPTION
**Pull request by changusmc**

### CODE REVIEW

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_PR-Check">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: libs/SalesforceSDK/build.gradle [hunk 1]
```diff
@@ -18,6 +18,7 @@ dependencies {
     api 'com.squareup.okhttp3:okhttp:3.9.0'
     api 'com.google.android.gms:play-services-gcm:11.8.0'
     api 'com.android.support:customtabs:25.2.0'
```
Comment (brandonpage) [inline libs/SalesforceSDK/build.gradle]: <table data-meta="generated_by_SalesforceSDK">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/7e76c91d63aa9cd6655127901e5ce32158ce9b54/libs/SalesforceSDK/build.gradle#L20"></span>A newer version of com.android.support:customtabs than 25.2.0 is available: 27.1.0</td>
    </tr>
  </tbody>
</table>

File: libs/SalesforceSDK/build.gradle [hunk 2]
```diff
@@ -18,6 +18,7 @@ dependencies {
...
+    androidTestApi 'junit:junit:4.11'
```
Comment (brandonpage) [inline libs/SalesforceSDK/build.gradle]: <table data-meta="generated_by_SalesforceSDK">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/7e76c91d63aa9cd6655127901e5ce32158ce9b54/libs/SalesforceSDK/build.gradle#L21"></span>A newer version of junit:junit than 4.11 is available: 4.12</td>
    </tr>
  </tbody>
</table>

**Comment by brandonpage** [comment]
# Tests results for SalesforceSDK

<p align="right" data-meta="generated_by_SalesforceSDK">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by bhariharan** [comment]
Our tests are running fine on CircleCI. I don't think we need this dependency.

**Comment by bhariharan** [comment]
@changusmc Running fine locally as well with the latest from `dev` without this patch. I think the issue is specific to your environment.
``
./gradlew :libs:SalesforceSDK:connectedAndroidTest
Starting 154 tests on Pixel_2_API_27(AVD) - 8.1.0
Download https://jcenter.bintray.com/org/jacoco/org.jacoco.ant/0.7.4.201502262128/org.jacoco.ant-0.7.4.201502262128.pom
Download https://jcenter.bintray.com/org/jacoco/org.jacoco.agent/0.7.4.201502262128/org.jacoco.agent-0.7.4.201502262128.jarg.jacoco.ant-0.7.4.201502262128.pom
Download https://jcenter.bintray.com/org/jacoco/org.jacoco.ant/0.7.4.201502262128/org.jacoco.ant-0.7.4.201502262128.jar

BUILD SUCCESSFUL in 1m 12s
70 actionable tasks: 3 executed, 67 up-to-date
```

**Comment by changusmc** [comment]
👍

**Pull request closed without merging**